static/css/main.css

```diff
@@ -80,17 +91,21 @@ h2 {
 
 a {
     text-decoration: none !important;
-    color: white !important;
+    color          : white !important;
 }
 
 session-code {
     display: none;
 }
+
 #dummy {
     width: calc(100vw - 16px)
 }
 
-:read-only {
-    white-space: pre-wrap;
-    width: 100%;
-} 
\ No newline at end of file
+button:hover {
+    color: white;
+}
+
+i {
+    font-size: 24px;
+}
```

Comment:
no newline at end of file

Reply:
my autoformatter removes trailing newlines